fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/PlayerSession.java

```diff
@@ -88,6 +88,8 @@ public record PlayerDto(String name,
                             RankEnum rank,
                             ProfessionEnum profession,
                             int stamina,
+                            String god,
+                            int staminaWhenUnwounded,
```

Comment:
we'll need a rule such that stamina can never go above this value... but maybe in another PR were we tackle all game rules as a whole: it's getting scattered.